modin/experimental/pandas/io.py

```diff
@@ -191,7 +191,7 @@ def _read(**kwargs) -> DataFrame:
     General documentation is available in `modin.pandas.read_csv`.
 
     This experimental feature provides parallel reading from multiple csv files which are
-    defined by glob pattern. Works for local files only!
```

Comment:
To fully support remote buckets, `storage_options` parameter will be added in #3829